pkg/controller.v1/pytorch/initcontainer_test.go

```diff
@@ -34,6 +34,7 @@ func TestInitContainer(t *testing.T) {
 
```

Comment:
We may want to test the whole of initContainer, although that is out of scope in this PR.
So we can follow up on another PR.